src/seeding.jl

```diff
@@ -68,7 +66,7 @@ end
 #   choose an arbitrary subset as seeds
 #
 
-mutable struct RandSeedAlg <: SeedingAlgorithm end
+struct RandSeedAlg <: SeedingAlgorithm end
```

Comment:
Could you please also convert the comments above into the docstring for this and the other seeding algorithms?